src/postcss/avatar-card.css

```diff
@@ -41,4 +53,8 @@ $avatar-name-font-color: $color-black-5;
     }
 
   }
+
+  &--small {
```

Comment:
You might have left this here by accident, I suppose! Please remove this selector, if it's not used.